core-dev/packages/coq.8.7.dev/opam

```diff
@@ -15,8 +15,12 @@ build: [
     "-coqide" "no"
     "-debug"]
   [make "-j%{jobs}%"]
+  [make "-j%{jobs}%" "byte"]
```

Comment:
I think that @letouzey explicitly warned against this and suggested the preferred form was `make -j && make -j byte`.
But I'm not the Makefile expert and won't argue more if you think that what you did is fine.

Reply:
Yes, that's prone to race conditions on cmi files

Reply:
OPAM is doing first `make -j` and then `make -j byte`, not at the same time (otherwise it would do, in parallel, also `./configure`).

So I think the opam rules I wrote and the command Pierre suggests to use are really semantically equivalent.

Reply:
Indeed, sorry @gares I misread the diff. I think you are right and these are equivalent indeed.